silkworm/db/datastore/kvdb/history_range_in_period_query.hpp

```diff
@@ -113,7 +113,8 @@ struct HistoryRangeInPeriodQuery {
                         if (key_decoder.value.timestamp.value < ts_range.start) {
                             *skip_current_key = false;
                             return {};
-                        } else if (key_decoder.value.timestamp.value < ts_range.end) {
+                        }
+                        if (key_decoder.value.timestamp.value < ts_range.end) {
```

Comment:
I'm not a fan of this lint rule. It is not clear what does it improve for us. I feel that the code is less readable and less robust.